cmd/dp-filter-api/main.go

```diff
@@ -99,6 +108,8 @@ func main() {
 
 		cancel()
 
+		healthTicker.Close()
```

Comment:
Consider calling close on the ticker before closing the mongo session.

Reply:
moved